### PR TITLE
fix(cleanup): remove circle icons and modify text

### PR DESCRIPTION
**Pull request by dgutride**

### CODE REVIEW

**Comment by fabric8cd** [comment]
@dgutride fabric8/fabric8-ui:SNAPSHOT-PR-1908-1 fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1908-fabric8-ui.badger.fabric8.io

File: src/app/profile/cleanup/cleanup.component.html [hunk 1]
```diff
@@ -9,7 +9,7 @@ <h3 *ngIf="cleanupStatus === 'notstarted'">
         <strong>You are about to erase all of your OpenShift.io spaces</strong>, reverting your OpenShift.io environment to it's default state.
```
Comment (joshuawilson) [inline src/app/profile/cleanup/cleanup.component.html]: Do we still want this to call out "spaces". This line is bold. The changed one is less noticeable.

File: src/app/profile/cleanup/cleanup.component.html [hunk 2]
```diff
@@ -58,7 +58,7 @@ <h3 *ngIf="cleanupStatus !== 'notstarted'">Results</h3>
             <template #itemTemplate let-item="item" let-index="index">
               <div class="list-pf-content-wrapper">
                 <div class="list-pf-main-content">
-                  <span class="cleanup-status-icon {{item.statusIcon}}"></span>
+                  <span *ngIf="item.statusIcon" class="cleanup-status-icon {{item.statusIcon}}"></span>
```
Comment (joshuawilson) [inline src/app/profile/cleanup/cleanup.component.html]: The space name appears to be indented 1 extra space and therefore doesn't align with the username above.

Comment (dgutride) [inline src/app/profile/cleanup/cleanup.component.html]: Made a small less change...

File: src/app/profile/cleanup/cleanup.component.html (hunk 1 above)
Comment (dgutride) [inline src/app/profile/cleanup/cleanup.component.html]: There is a separate verbiage pass that is ongoing that might result in some other changes.  This layout matches the design as coded.

**Comment by fabric8cd** [comment]
@dgutride fabric8/fabric8-ui:SNAPSHOT-PR-1908-2 fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1908-fabric8-ui.badger.fabric8.io

File: src/app/profile/cleanup/cleanup.component.html (hunk 2 above)
Comment (joshuawilson) [inline src/app/profile/cleanup/cleanup.component.html]: It's still not straight to my eye, but I'll let @mindreeper2420 fix it in another PR.

**Pull request merged**